core/shared/src/main/scala/scalaz/zio/internal/Stack.scala

```diff
@@ -19,7 +19,7 @@ package scalaz.zio.internal
 /**
  * A very fast, growable/shrinkable, mutable stack.
  */
-final class Stack[A <: AnyRef]() {
+private[zio] final class Stack[A <: AnyRef]() {
```

Comment:
Don't forget `StackBool` while you're at it!

Reply:
Right, I've thought that private constructor is sufficient :(

Reply:
If the class itself is not private, it will appear in the generated docs... 😦